regions/core/core.py

```diff
@@ -283,47 +259,29 @@ def union(self, other):
         from .compound import CompoundSkyRegion
         return CompoundSkyRegion(self, other, operator.or_)
 
-    @abc.abstractmethod
-    def contains(self, skycoord):
+    def contains(self, skycoord, wcs):
```

Comment:
Can we add ``__contains__`` calling this method also for SkyRegions (it's there for pixel regions)